proto/rill/admin/v1/api.proto

```diff
@@ -1274,6 +1301,34 @@ message SudoIssueRuntimeManagerTokenResponse {
   string token = 1;
 }
 
+message SudoDeleteOrganizationBillingErrorRequest {
+  string organization = 1;
+  BillingErrorType type = 2;
+}
+
+message SudoDeleteOrganizationBillingErrorResponse {}
+
+enum BillingErrorType {
+  BILLING_ERROR_TYPE_UNSPECIFIED = 0;
+  BILLING_ERROR_TYPE_NO_PAYMENT_METHOD = 1;
+  BILLING_ERROR_TYPE_NO_BILLABLE_ADDRESS = 2;
+  BILLING_ERROR_TYPE_INVOICE_PAYMENT_FAILED = 3;
+  BILLING_ERROR_TYPE_TRIAL_ENDED = 4;
+  BILLING_ERROR_TYPE_SUBSCRIPTION_CANCELLED = 5;
+}
```

Comment:
By convention, non-request/response types are placed at the bottom of the file after all the request/response types